docs/_archive/bugs/BUG-046-datetime-serialization-news-collector.md

```
@@ -0,0 +1,70 @@
+**Priority:** High
+**Discovered:** 2026-01-09
+**Resolved:** 2026-01-09
+**Component:** `news/collector.py`, `exa/client.py`
+
+---
+
+## Symptom
+
+When running `kalshi news collect`, the following error occurs:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)